src/networking.c

```diff
@@ -3942,6 +3942,13 @@ void processEventsWhileBlocked(void) {
      * interaction time with clients and for other important things. */
     updateCachedTime(0);
 
+    /* For the few commands that are allowed during busy scripts, we rather
+     * provide a fresher time than the one from when the script started (they
+     * still won't get it from the call due to execution_nesting. For commands
```

Comment:
WDYM by "they still won't get it from call"? won't get what? fresher time?
maybe "(execution_nesting is >=1 for all commands that run during a busy script, so cmd_time_snapshot won't be updated inside `call` until the script is done)"

Reply:
yes, they won't get a fresher time.
i think your explanation is longer and more technical will be harder to read.
we just need to hint that it's still an inaccurate compromise, and people can read the code to understand it better.
i can replace "it" with "fresher time" if that helps.

let's first discuss the solution and later the comment..